README.md

```diff
@@ -17,6 +17,7 @@ The versions used in the latest version are:
 
 * Robot Framework 3.0.2
 * Robot Framework SeleniumLibrary 3.0.1
+* Robot Framework Faker 4.2.0
```

Comment:
Could you create a link to the Robot Faker project please? https://github.com/guykisel/robotframework-faker